### PR TITLE
:mortar_board: Topic ALU --- Show running program examples 

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Show two examples of running programs on the ALU

Also remind them that hex is a thing

Warn about being careful about the state of their system

### Testing

:+1:

### CODE REVIEW

File: site/topics/alu/esap-alu.rst [hunk 1]
```diff
@@ -77,6 +77,249 @@ Design of the ESAP ALU
...
+
+
+* Below is a table showing how to add the numbers 15 and 4 together with the ALU
+
+    * Like with the bus, this is not so much a truth table, but a program
+    * ``C`` means clock pulse
+    * ``Z`` is a high impedance state
+
+
+.. list-table:: Program to add 15 and 4 together
```
Comment (jameshughes89) [inline site/topics/alu/esap-alu.rst:225]: image

File: site/topics/alu/esap-alu.rst [hunk 2]
```diff
@@ -77,6 +77,249 @@ Design of the ESAP ALU
...
+    #. Output from register A
+
+
+* Below is a table representing the above program
+* Notice how, in this example, several steps are able to be performed in a single clock pulse
+
+    * Namely, setting subtraction, outputting from the ALU, and inputting into A
+
+
+.. list-table:: Program to subtract 15 and 4 and store the result in A, then output the final result
```
Comment (jameshughes89) [inline site/topics/alu/esap-alu.rst:315]: image

File: site/topics/alu/esap-alu.rst (hunk 1 above)
Comment (twentylemon) [reply site/topics/alu/esap-alu.rst:225]: Isn't 15 `1111` ? Why is this `10000`?

Comment (twentylemon) [reply site/topics/alu/esap-alu.rst:225]: Wait. Wrong registers. Sleepy.

**Review by twentylemon — APPROVED** [review]

**Pull request merged**